### ISSUE TITLE
Upgrade/replace Chokidar and FSEvents

### ISSUE THREAD

**Issue by Chapabu** [frctl/fractal#585]
Whilst installing Fractal on a new project, I noticed the following notice:

![image](https://user-images.githubusercontent.com/1395471/81028438-5109b880-8e79-11ea-9ab4-1fbdac5c73a1.png)

I'm _pretty sure_ fsevents is a security issue due to something about insecure binaries on AWS. I'm not 100% sure of the ins and outs - maybe mentioning @paulmillr will work in a repo he's not part of :crossed_fingers:.

Node 14 it going to become the new LTS shortly, so it's imperative that Fractal works on it.

I'll take this one on, as it feels like a good opportunity to work out some of the kinks and dodgy old code at the same time.

**Comment by paulmillr** [comment]
Yes, important to upgrade

**Comment by mihkeleidast** [comment]
I took a look at this today since I think @Chapabu has not been able to work on this yet.

FWIW this is not a very high priority for our users since the only dependency that used chokidar 2 was `browser-sync` and they've since updated chokidar in a patch version, which means that the users will get the updated chokidar/fsevents anyway.

However, this is going to stop us from updating our dev environment to Node 14.  chokidar < 3 is used in our deps by gulp -> glob-watcher and watchify. Neither of those dependencies have a version out that has the update in. We use those deps for developing/bundling mandelbrot. I'll propose moving away from gulp for that and to a more modern bundling setup. I'd personally go for webpack since that's what I'm most familiar with (also I already did that for my employer's fork of mandelbrot, so that'll help), but if anyone else wants to take this on we can go another route.

I'll file a PR for updating browser-sync & chokidar to latest for our own lockfile so we can get rid of a part of the problem, but the mandelbrot bundling setup will be a bit bigger task.
